### PR TITLE
Create percentage of functional

### PR DESCRIPTION
**Pull request by esoliani**

### CODE REVIEW

File: app/scripts/components/dashboard/charts/waterpoint-functional-chart.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        };
+      }).sort(comparator),
+    }];
+  },
+
+  render() {
+    const waterpointsRes = func.Result.countByGroupBy(this.props.waterpoints, 'REGION', 'STATUS');
+    return (
+      <div className="waterpoint-functional-chart">
+        <BarChart
```
Comment (uniphil) [inline app/scripts/components/dashboard/charts/waterpoint-functional-chart.jsx]: Wrap `<BarChart />` with `<TSetChildProps>` to translate it, like 

``` jsx
<TSetChildProps><BarChart title={{k='chart.functional-waterpoints'}} ... /></TSetChildProps>
```

File: app/scripts/components/dashboard/charts/waterpoint-status-chart.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ const StackBarChart = React.createClass({
   },
 
   render() {
-    const dataRes = func.Result.countByGroupBy(this.props.data, 'STATUS', 'REGION');
+    const dataRes = func.Result.countByGroupBy(this.props.waterpoints, 'STATUS', 'REGION');
     return (
       <div className="stack-bar-chart">
         <BarChart
```
Comment (uniphil) [inline app/scripts/components/dashboard/charts/waterpoint-status-chart.jsx]: Wrap with child props translator

**Comment by uniphil** [comment]
Awesome, this is looking great.

I'm not sure the best way to translate the database column names that show up in the chart yet. Open to ideas, but we can leave it for now to not block this code from landing.

**Comment by uniphil** [comment]
Closed for translation change requests

**Comment by esoliani** [comment]
review comment fixed

**Comment by uniphil** [comment]
Arg, my child-props-overriding translate component doesn't work with React's prop typechecking -- it gives warnings for the objects that should be strings, even though they are turned into strings...

I guess it can use a refactor. Shouldn't block this PR though.

**Pull request merged**